mods/chesttools/depends.txt

```diff
@@ -1 +0,0 @@
-default
```

Comment:
Chest tools actually depends on default. I get the following error when trying to boot
```
2019-10-05 15:35:31: ERROR[Main]: ModError: Failed to load and run script from /home/minetest/.minetest/worlds/world/worldmods/chesttools/init.lua:
2019-10-05 15:35:31: ERROR[Main]: ...est/.minetest/worlds/world/worldmods/chesttools/init.lua:381: attempt to index global 'default' (a nil value)
```